src/Sarif.Multitool/AbsoluteUriCommand.cs

```diff
@@ -46,7 +46,7 @@ private static IEnumerable<AbsoluteUriFile> GetSarifFiles(AbsoluteUriOptions abs
             HashSet<string> fileNames = MultitoolFileHelpers.CreateTargetsSet(absoluteUriOptions.TargetFileSpecifiers, absoluteUriOptions.Recurse);
             foreach (var file in fileNames)
             {
-                yield return new AbsoluteUriFile() { FileName = file, Log = MultitoolFileHelpers.ReadSarifFile(file) };
+                yield return new AbsoluteUriFile() { FileName = file, Log = MultitoolFileHelpers.ReadSarifFile<SarifLog>(file) };
```

Comment:
I see that in MultitoolFileHelper.cs, you changed the API so that the generic parameter in the call to `JsonConvert.DeserializeObject`, which was formerly encapsulated, is now exposed to the caller. What is the advantage of this?

Reply:
Never mind, I see that it's so you can use it for `SarifLogVersionOne` as well.